app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,140 @@
+class ProjectsController < ApplicationController
+  load_and_authorize_resource
+  def index
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def create
+    company = Company.first
+    @project.company = company
+
+    if @project.save
+      flash[:success] = t('flash_messages.create', name: 'Project')
+    else
+      flash[:danger] = t('flash_messages.error', error_msg: @project.errors.full_messages.first)
+    end
+
+    respond_to do |format|
+      format.js { redirect_to projects_path }
+    end
+  end
+
+  def edit
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def update
+    if @project.update(project_params)  
+      flash[:success] = t('flash_messages.update', name: 'Project')
+    else
+      flash[:danger] = t('flash_messages.error', error_msg: @project.errors.full_messages.first)
+    end
+    respond_to do |format|
+      format.js { redirect_to projects_path }
+    end
+  end
+
+  def show
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def destroy
+    if @project.destroy
+      flash[:success] = t('flash_messages.destroy', name: 'Project')
+    else
+      flash[:warning] = t('flash_messages.warning', warning_msg: "Project can't be Destroyed")
+    end
+    respond_to do |format|
+      format.html { redirect_to projects_path }
+    end
+  end
+
+  def project_users
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def new_team
```

Comment:
Does this method create new resource? If not, does naming sound right? Please verify **everywhere** for such ambiguous method names.